src/api/Shrooms.Contracts/Enums/LotteryStatus.cs

```diff
@@ -5,9 +5,10 @@ public enum LotteryStatus
         Drafted = 1,
         Started,
         Deleted,
-        Ended,
+        Finished, // User finished the lottery

```

Comment:
I would say it's still confusing - "ended" and "finished" are synonyms, and you still need to have comments to explain it. :)
"Time ran out" - I would say it could be "Expired". Perhaps it would a bit negative meaning (that it finished unintentionally), but at least it would be less confusing, than having both "ended" and "finished".

(I know you did it like suggested in the backlog issue, but perhaps it was needing a better grooming.)